submissions/AnnaStoyano/Friends-app/index.js

```diff
@@ -0,0 +1,163 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const API_PATH = 'https://randomuser.me/api/';
+    const USER_COUNT = 20;
+    const URL_FOR_SEARCH = `${API_PATH}?results=${USER_COUNT}`;
+
+    fetch(URL_FOR_SEARCH)
+        .then(resolve => resolve.json())
+        .then(data => {
+            const friendsApp = new FriendsApp(data.results);
+            friendsApp.start();
+        });
+});
+
+class FriendsApp {
+    constructor(users) {
+        this.resultsWrapper = document.querySelector('.result');
+        this.cards = users.map(item => new UserCard(item));
+        this.dipslayCards = this.cards;
+        this.previouseDisplay = this.dipslayCards;
+        this.filterByName();
+        this.filterByAge();
+        this.filterByGender();
+        this.filterByCity();
+        this.filterByEmail();
+    }
+
+    start() {
+        this.render(this.dipslayCards);
+    }
+
+    render() {
+        this.resultsWrapper.innerHTML = '';
+        this.dipslayCards.forEach(({card}) => this.resultsWrapper.insertAdjacentElement('afterbegin', card));
+    }
+
+    filterByName() {
+        this.filterByNameAlphabet();
+        this.filterByNameNonAlphabet();
+    }
+
+    filterByNameAlphabet() {
+        const alphabetBtn = document.querySelector('.name-alphabetically');
+        alphabetBtn.addEventListener('click', () => {
+            this.dipslayCards = this.dipslayCards.sort((a, b) => a.name > b.name ? -1 : 1);
+            this.render();
+        });
+    }
+
+    filterByNameNonAlphabet() {
+        const nonAlphabetBtn = document.querySelector('.name-nonalphabetically');
+        nonAlphabetBtn.addEventListener('click', () => {
+            this.dipslayCards = this.dipslayCards.sort((a, b) => a.name > b.name ? 1 : -1);
+            this.render();
+        });
+    }
```

Comment:
Why do you need these two functions? They look the same

Reply:
First sort names alphabet alphabetical. Second sort names not alphabetical